cetmix_tower_server/models/cx_tower_template_mixin.py

```diff
@@ -17,6 +17,29 @@ class CxTowerTemplateMixin(models.AbstractModel):
 
     code = fields.Text(string="Code", help="This field will be rendered by default")
 
+    variable_ids = fields.Many2many(
+        comodel_name="cx.tower.variable", compute="_compute_variable_ids", store=True
+    )
+
+    @api.depends("code")
+    def _compute_variable_ids(self):
```

Comment:
We should keep in mind that variables can be used not in the `code` field only.
For example they are used in:
- [Command path](https://github.com/cetmix/cetmix-tower/blob/047ba72ed62f2b0a7eb669b71daa22e028312f3d/cetmix_tower_server/models/cx_tower_command.py#L94)
- Flight plan line [condition](https://github.com/cetmix/cetmix-tower/blob/047ba72ed62f2b0a7eb669b71daa22e028312f3d/cetmix_tower_server/models/cx_tower_plan_line.py#L45)
- [Variable value](https://github.com/cetmix/cetmix-tower/blob/047ba72ed62f2b0a7eb669b71daa22e028312f3d/cetmix_tower_server/models/cx_tower_variable_value.py#L25)
- [File template](https://github.com/cetmix/cetmix-tower/blob/14.0-dev/cetmix_tower_server/models/cx_tower_file_template.py)
  - file_name
  - server_dir
  - code
  
- [File](https://github.com/cetmix/cetmix-tower/blob/14.0-dev/cetmix_tower_server/models/cx_tower_file.py)
  - name
  - server_dir
  - code